documentation/src/main/asciidoc/ch11.asciidoc

```diff
@@ -424,7 +424,7 @@ unwrapped to `HibernateConstraintViolation` in order to retrieve the dynamic pay
 ====
 [source, JAVA, indent=0]
 ----
-include::{sourcedir}/org/hibernate/validator/referenceguide/chapter11/dynamicpayload/DynamicPayloadTest.java[tags=include]
+include::{sourcedir}/org/hibernate/validator/referenceguide/chapter11/dynamicpayload/DynamicPayLoadTest.java[tags=include]
```

Comment:
Ah, I fugure that one was broken? I just closed [HV-1046](https://hibernate.atlassian.net/browse/HV-1046) beause I couldn't it, but it seems that's the one? If so, let's re-open and reference it then.